src/routes/safe/components/Transactions/TxsTable/ExpandedTx/index.tsx

```diff
@@ -57,10 +58,20 @@ const ExpandedTx = ({ cancelTx, tx }) => {
         <Row>
           <Col layout="column" xs={6}>
             <Block className={cn(classes.txDataContainer, (isIncomingTx || isCreationTx) && classes.incomingTxBlock)}>
-              <Block align="left" className={classes.txData}>
```

Comment:
are styles for this classname are used somewhere else? if no, you can remove them

Reply:
not the whole file, but I removed unsued classes.